app/controllers/api/v1/apps/credentials_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Api::V1::Apps::CredentialsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read }
+
+  respond_to :json
+
+  def show
+    render json: doorkeeper_token.application, serializer: REST::StatusSerializer::ApplicationSerializer
```

Comment:
maybe we need to move this out of StatusSerializer if it's now shared?

Reply:
If there's gonna be a 3rd place where this is used, then I'll agree. At 2, not really worth it.